aten/src/ATen/Parallel.cpp

```diff
@@ -51,4 +52,5 @@ void init_tbb_num_threads() {
     num_threads_ = num_threads;
   }
 }
-}} // namespace at::internal
+} // namespace internal
+} // namespace at
```

Comment:
Can you please avoid changes like those? It's not even consistent with what we usually do

Reply:
It's using the clang project linter. I can't remember all the conventions. We can edit that config.